### DISCUSSION TITLE
这个repo和深度学习、机器学习类的各种awesome的边界在哪？ 以及其他问题

### DISCUSSION THREAD

**Discussion by lgh06** [xcatliu/awesome-aigc#2] [General]
follow很久了 偶然刷到的。
最近我在玩ai作图，所以想加入讨论。

1. 这个repo和深度学习、机器学习类的各种awesome的边界在哪……？
2. 中文？英文？资源是否要明确分开？
3. 比如百度的 https://aistudio.baidu.com/ ，算是个和google colab类似的工具，所以也许应该有一个在线工具的section 吧…
4. 各种已经商业化的ai作图、文本生成语音等服务，是否算aigc领域，是否在这个repo的范畴内？ 是否也有一个专门的section…

Waiting your reply.
thx.

**Comment by xcatliu** [comment]
1. 由于我不是ai相关专业，所以对核心算法理解不够，我更关注应用层相关的工具
2. 更倾向英文资源，会更丰富。但也接受中文资源
3. 嗯，接受工具章节
4. 已商业化的也接受，商业化是可持续发展的标志
